src/main/resources/application.yml

```diff
@@ -83,3 +83,8 @@ logging:
     root: INFO
   pattern:
     console: "%clr(%d{dd-MM-yyyy HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr([%35.35t]){faint} %clr(%-28.28logger{28}){cyan} %clr(:){faint}%X{BUSINESS-LOG} %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}"
+
+gsma:
+  payee:
+    tenant: "lion"
```

Comment:
this needs to be redesigned to move this param to request specific or workflow specific